### PR TITLE
use LinkHandler to resolve t3:// links

### PR DESCRIPTION
**Pull request by bernhardberger**

### CODE REVIEW

File: Classes/ContentObject/MediaContentObject.php [hunk 1]
```diff
@@ -107,10 +107,13 @@ public function render($conf = array())
             // Default renderType is swf
             $renderType = 'swf';
             $handler = array_keys($conf['fileExtHandler.']);
+
+            $linkService = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\LinkHandling\LinkService::class);
```
Comment (merzilla) [inline Classes/ContentObject/MediaContentObject.php:111]: Thanks a lot, @bernhardberger 
Currently the LinkService is only needed in line 114 and 116. But what do you think?
Would you like to move the line 111 at the beginning of the function? To line 31 or so?
So the initialization would be on top and the `$linkService` could be used somewhere else here, maybe in future somewhen.

**Review by merzilla — COMMENTED** [review]
Hi @bernhardberger 

please have a look at my suggestions.

Thx

**Comment by bernhardberger** [comment]
Why not..

I just put it down there inside the ifs so it's only loaded when actually needed to reduce memory/performance overhead

**Comment by merzilla** [comment]
Well ok... that's a point, too. So we keep it.

**Pull request merged**